onnxruntime/test/providers/cpu/tensor/split_op_test.cc

```diff
@@ -264,6 +264,18 @@ ShapeAndFloatData CreateInput(std::vector<int64_t> shape) {
   return input;
 }
 
+template <typename T>
```

Comment:
Can't CreateInput be replaced by this? Your version would handle the float case too?

Reply:
Yes, should be.